javascript/javascript_sdk/src/get_data.js

```diff
@@ -0,0 +1,85 @@
+const google_protobuf_timestamp_pb = require('google-protobuf/google/protobuf/timestamp_pb.js');
+
+const get_zone_data = (client) => {
+    client.listenToObjectUpdate(response => {
+      let zone_lists = response.getZonesList();
+      
+      zone_lists.forEach(zone_list => {
+          let obj = zone_list.getObject();
+          if (zone_list.getType() == 1) {
+              console.log("Enter zone(", zone_list.getId() ,") : ", obj.getId());
+          }
+          else if (zone_list.getType() == 2) {
+              console.log("Exit zone(", zone_list.getId() ,") : ", obj.getId());
+          }
+      });
+    });
+  }
+  
+const get_point_data = (client) => {
+    client.listenToPointCloudUpdate(response => {
+        const pointClouds = response.getPointsList();
+        
+        pointClouds.forEach(pointCloud =>{
+            let pointcloud_info = pointCloud.array;
+            
+            let num_points = Math.floor((pointcloud_info[2].length)/(3.0));
```

Comment:
Don't we need to do `Math.floor((pointcloud_info[2].length)/(3.0 * sizeof(float)))` here? I'm not sure how you get the size of float in javascript, but you get the point.